android/build.gradle

```diff
@@ -144,26 +140,39 @@ repositories {
     mavenCentral()
 }
 
-def kotlin_version = getExtOrDefault('kotlinVersion', project.properties['RNSAC_kotlinVersion'])
```

Comment:
@KrzysztofMoch can you please check this change ? I think it breaks your last patch ?